### DISCUSSION TITLE
Can't access C enum

### DISCUSSION THREAD

**Discussion by prime31** [vlang/v#11627] [Ideas]
**V version:** V 0.1.23 fc64238
**OS:** macOS 10.15.2 (19C57)

**What did you do?**
```c
// C enum
enum Thing {
	First,
	Second
}
```

```c
// V declaration
enum C.Thing { First, Second }
```

Attempting to access a C enum results in type undefined compiler errors.

**What did you expect to see?**
I expected to be able to define and access the C enum
 
**What did you see instead?**
Missing type errors on compile

**Comment by ghost** [comment]
Same problem

**Comment by KeitoTobi1** [comment]
Same Problem.

version: V 0.2.4 `1370516`
**OS**: Windows 11 Pro 21H1

test/thing.c
```c
enum Thing {
    First,
    Second
}
```

test/thing.c.v
```v
enum C.Thing {
    First
    Second
}
```

v fmt log:
```
test\thing.c.v:1:6: error: single letter capital names are reserved for generic template types.
    1 | enum C.Thing {
      |      ^
    2 |     First
    3 |     Second
Internal vfmt error while formatting file: .\test\thing.c.v.
```
